### PR TITLE
order detail

### PR DESCRIPTION
**Pull request by dungpt-0371**
## Related Tickets:
- [#69064](https://edu-redmine.sun-asterisk.vn/issues/69064)

## WHAT this PR does?
- Danh sách đơn hàng từ trước đến nay
- Thông tin chi tiết đơn hàng

## Notes Impacted Areas:
- app/Http/Controllers
- routes/web.php
- app/Models
- resources/views/orders
- resources/views/orderDetails

## Attachments:
image
image

### CODE REVIEW

**Comment by dungpt-0371** [comment]
ready

**Comment by hieupc-0248** [comment]
oke

File: resources/views/orderDetails/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                <tbody class="bg-gray-50">
+                    @foreach ($order->orderDetails as $index => $order_detail)
+                        <tr class="border-b-4">
+                            <td class="text-gray-900 text-center py-4">{{++$index}}</td>
+                            <td class="text-gray-900 text-center py-4"><a href="/books/{{ $order_detail->book->id }}">
+                                    {{ $order_detail->book->name }}
+                                </a></td>
+                            <td class="text-gray-900 text-center py-4">{{ $order_detail->quantity }}</td>
+                            <td class="text-gray-900 text-center py-4">$ {{ $order_detail->book->price }}</td>
+
```
Comment (ghost) [inline resources/views/orderDetails/index.blade.php]: delete

File: app/Http/Controllers/OrderController.php [hunk 2]
```diff
@@ -62,7 +76,8 @@ public function store(Request $request)
      */
     public function show(Order $order)
     {
-        //
+        // dd($order);
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]: delete

File: app/Http/Controllers/OrderController.php [hunk 3]
```diff
@@ -47,11 +51,21 @@ public function store(Request $request)
...
+            $orderDetail->order_id = $order->id;
+            $orderDetail->save();
+        }
+
         $carts = Cart::where('user_id', $user->id)->get();
 
         DB::table('carts')->whereIn('id', $carts->pluck('id'))->delete();
 
-        return redirect('/carts');
+        return redirect('/orders');
```
Comment (ghost) [inline app/Http/Controllers/OrderController.php]: use route()

**Comment by dungpt-0371** [comment]
ready

**Comment by dungpt-0371** [comment]
ready

**Pull request merged**